### DISCUSSION TITLE
Generating the Pydantic Output to a New File

### DISCUSSION THREAD

**Discussion by geob3d** [tiangolo/pydantic-sqlalchemy#90] [Questions] [answered]
Hi Tiangolo 

I was wondering if there was a way to generate a new py file from the pydantic-sqlalchemy converter. That would be a great feature to add that component to your script but I may have not figure it out yet.

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile:

I deprecated this library a while ago in favor of SQLModel which would solve all the same use cases in a much better way, and I'm directing all the effort in that direction.

SQLModel uses Pydantic models to define both data and table (SQL) models, so it's easier to do all these things. :rocket: 

In fact, there's no need to output anything to a file because you start with something that is both a SQLAlchemy model and a Pydantic model at the same time. :sunglasses:
